### ISSUE TITLE
Accordions `aria-disabled` issue

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#569]
https://modus-web-components.trimble.com/?path=/docs/components-accordion--default

![image](https://user-images.githubusercontent.com/1212885/181173999-81bea2da-8bcb-4c8b-8c61-fff9fb6ca9c8.png)

`aria-disabled` should be `aria-disabled="true"`

**Comment by coliff** [comment]
heya @ryan-henness-trimble - if you get a chance can you take a look at this? I think it's likely pretty easy if you know the code better than I do. Just need to change `aria-disabled` to `aria-disbled="true"`.

**Comment by ryan-henness-trimble** [comment]
hmm, this should only be disabled if the item is disabled, right?

**Comment by coliff** [comment]
> hmm, this should only be disabled if the item is disabled, right?

correct
